.vscode/launch.json

```diff
@@ -0,0 +1,34 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "lldb",
+      "request": "launch",
+      "name": "Debug Moonbeam Node",
+      "cargo": {
+        "args": ["build", "--bin=moonbeam"],
```

Comment:
isn't it supposed to be cargo run and not cargo build ?

Reply:
CodeLLDB supports a subset of cargo commands, and expects something that builds binaries and don't run them. So `build` or `test --no-run`.

Reply:
👍